src/fileformats/ocd_file_export.cpp

```diff
@@ -175,12 +176,24 @@ bool isStructureShiftedRows(const AreaSymbol::FillPattern& first, const AreaSymb
 }
 
 
+/**
+ * Test if the fill patterns consitute OCD structure "aligned rows".
+ */
+bool isStructureAlignedRows(const AreaSymbol::FillPattern& first, const AreaSymbol::FillPattern& second) noexcept
+{
+	return first.point_distance == second.point_distance
+	       && first.line_spacing == second.line_spacing
+	       && qFuzzyCompare(first.angle, second.angle)
+	       && first.rotatable() == second.rotatable();
+}
+
 
 /**
- * Postprocess a fill pattern symbol for optimal OCD structure export.
+ * Postprocess a fill pattern sequence for optimal OCD structure export.
  */
-std::unique_ptr<PointSymbol> postProcessFillPattern(const AreaSymbol::FillPattern* first_point_pattern)
+std::unique_ptr<PointSymbol> postProcessFillPattern(const AreaSymbol::FillPattern** first, const AreaSymbol::FillPattern** last, int structure_mode)
```

Comment:
No need to use iterator-style pointers here. I will change this to `SomeContainterTypeAlias const &` tonight.

Reply:
Done.